en/guide/offboard.md

```diff
@@ -0,0 +1,193 @@
+The direction is specified in clockwise degrees relative to North (0 is North, 90 is East, 180 is South, etc.)
+
+Calling `set_velocity_ned()` using an initialiser list type declaration for the `VelocityNEDYaw` argument, 
+the final (fourth) value is the yaw direction.
+
+Examples:
+* Turn to face West:
+  ```cpp
+  device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 270.0f});
+  ```
+* Turn to face North:
+  ```cpp
+  device.offboard().set_velocity_ned({0.0f, 0.0f, 0.0f, 0.0f});
+  ```
+
+It is not possible to control the rate or direction that the vehicle will use to turn towards the setpoint direction (it will turn in whatever direction reaches the setpoint fastest).
+  
+
+### Turn/Yaw Vehicle in specified Direction/at Rate
+
+The `set_velocity_body()` can be used to rotate the vehicle at a specific rate and in a specified direction. 
+This is set in [VelocityBodyYawspeed::yawspeed_deg_s](../api_reference/structdronecore_1_1_offboard_1_1_velocity_body_yawspeed.md#structdronecore_1_1_offboard_1_1_velocity_body_yawspeed_1a6858130475964eb2d5c5a4236b7f1e31), as the angular rate in degrees/second. If viewed from above, the vehicle will turn clockwise if the value is positive and anticlockwise if it is negative. 
+
+Calling `set_velocity_body()` using an initialiser list type declaration the final (fourth) value is the yaw rate/direction.
+
+Examples:
+
+* Turn clock-wise at 60 degrees per second:
+  ```cpp
+  device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, 60.0f});
+  ```
+* Turn anti clock-wise at 5 degrees per second:
+  ```cpp
+  device.offboard().set_velocity_body({0.0f, 0.0f, 0.0f, -5.0f});
+  ```
+  
+### Fly Forwards 
+
+Use `set_velocity_body()` to set the velocity components relative to the body frame. To fly forwards, simply set the first parameter (`Offboard::VelocityBodyYawspeed::forward_m_s`) when the vehicle is not rotating.
+
+```cpp
+device.offboard().set_velocity_body({5.0f, 0.0f, 0.0f, 0.0f});
+```
+
+### Fly a Circle
+    
+To fly a circle, use `set_velocity_body()` with both forward and rotational components. This will force the vehicle to travel in a curved path.
+
+```cpp
+device.offboard().set_velocity_body({5.0f, 0.0f, 0.0f, 30.0f});
+```
+
+You can force the vehicle to fly sideways by using the (`Offboard::VelocityBodyYawspeed::right_m_s` value), and in the other direction by using a negative rotation value:
+```cpp
+// Fly a circle sideways
+device.offboard().set_velocity_body({0.0f, -5.0f, 0.0f, -30.0f});
+```
+
+
+## Position/Thrust Setpoints
+
+DroneCore does not support position or thrust setpoints (at time of writing).
+
+
+## Waiting on Setpoints
+
+The vehicle will obey the last setpoint called (when you call a setpoint the last one is cleared). 
+The DroneCore examples use timers to separate commands (e.g. `sleep_for(seconds(8))`) but in a
+real-world use case you might use telemetry or sensors to control when the setpoint is changed.
+
+
+## Monitoring Offboard Mode

```

Comment:
@julianoes FYI, I added this "specific" note that vehicles potentially having multiple command sources in offboard mode. This is merged now.